mpi/Makefile

```diff
@@ -9,8 +9,9 @@ GENCODE_SM37	:= -gencode arch=compute_37,code=sm_37
 GENCODE_SM50	:= -gencode arch=compute_50,code=sm_50
 GENCODE_SM52	:= -gencode arch=compute_52,code=sm_52
 GENCODE_SM60    := -gencode arch=compute_60,code=sm_60
-GENCODE_SM70    := -gencode arch=compute_70,code=sm_70 -gencode arch=compute_70,code=compute_70 
-GENCODE_FLAGS	:= $(GENCODE_SM60) $(GENCODE_SM70)
+GENCODE_SM70    := -gencode arch=compute_70,code=sm_70 -gencode arch=compute_70,code=compute_70
```

Comment:
Applies to all changes: Please drop `-gencode arch=compute_70,code=compute_70` from `GENCODE_SM70` for forward compatibility it is sufficient to generate PTX (`code=compute_70`) for the latest architecture. See https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#application-compatibility

Reply:
Done.